ui/admin/app/routes/scopes/scope/targets/target/add-brokered-credential-sources.js

```diff
@@ -40,11 +40,11 @@ export default class ScopesScopeTargetsTargetAddBrokeredCredentialSourcesRoute e
     await all(
       credentialStores.map(({ id: credential_store_id, isStatic }) => {
         if (isStatic) {
-          return this.store.query('credential', {
+          this.store.query('credential', {
```

Comment:
I'm slightly confused, if you don't return the query we're not getting promises back for the array on the `Promise.all` so what will it be awaiting on?

(Unrelated but do we need to use rsvp `all` instead of `Promise.all`?)

Reply:
Update: After an offline investigation, we found out that this could be a side effect of the ember-data upgrade, but due to time constraints, we are going to merge this fix as is, which is consistent with other instances where peekAll() is used and doesn't throw an error. This will be added to the backlog for further investigation.

Reply:
None blocking comment: still confuse about the `await` Id o believe we need to `await` when we are using `store.query` to wait for the results?

Reply:
yes, we will need to await on the query method, sorry for the confusion regarding my comment.